src/PersonalSite.Tests/PersonalSite.UnitTests/FakeInMemoryPersonalSiteDbContext.cs

```diff
@@ -1,26 +1,19 @@
 ﻿using Microsoft.EntityFrameworkCore;
-using PersonalSite.Persistence.Mappings;
 using PersonalSite.Domain.Model.JobExperienceAggregate;
+using PersonalSite.Persistence;
 
 namespace PersonalSite.UnitTests;
 
-public class FakeInMemoryPersonalSiteDbContext : DbContext
+public class FakeInMemoryPersonalSiteDbContext : PersonalSiteDbContext
 {
-    public DbSet<JobExperience> JobExperiences { get; set; }
-
-    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+    public FakeInMemoryPersonalSiteDbContext(DbContextOptions<PersonalSiteDbContext> options) : base(options)
     {
-        optionsBuilder.UseInMemoryDatabase(databaseName: "FakePersonalSiteDbContext");
     }
 
-    protected override void OnModelCreating(ModelBuilder modelBuilder)
-    {
-        base.OnModelCreating(modelBuilder);
-        AddPersonalSiteMappings(modelBuilder);
-    }
+    public DbSet<JobExperience> JobExperiences { get; set; }
```

Comment:
Please review the warnings in the github action run and judge yourself if you need to change anything